README.md

```diff
@@ -1,2 +1,41 @@
 # Mentoring-Bot
 A discord bot for the end of time
+
+## Setup
+* Clone the repo to your local computer
+* Create a `.env` file. This will store the data the bot needs for execution
+  * See the header below for structure to the `.env`
+* Ensure both `node.js` and `npm` are installed on your system.
+* Run `npm install` to install all of the proper packages for the bot
+* Reach out to the current tech head to be added to the test channel for the
+  bot
+
+## .Env File
+This file is used to hold all of the important information in regards to the
+bot id and channel ids for execution. Currently there are 4 values that need
+to be defined:
+* BOT_TOKEN -> The identification token of the bot. Discuss with the current
+  tech head for information on what the test bot's id is. NEVER share the bot
+  token to anyone you don't trust
+* BOT_ID -> The identification number of the bot. If setup properly,
+  this value can be obtained by right clicking on the channel and selecting
```

Comment:
Switch channel to bot

Reply:
Good catch, adjusted here to say bot, and then channel or channel header for the following lines